src/test/java/com/nimbusds/jose/crypto/impl/XC20PTest.java

```diff
@@ -0,0 +1,74 @@
+package com.nimbusds.jose.crypto.impl;
+
+/*
+ * nimbus-jose-jwt
+ *
+ * Copyright 2012-2016, Connect2id Ltd and contributors.
```

Comment:
I believe the year needs to be `2012-2021`

Reply:
yeah i will fix documentation